src/components/Preview.js

```diff
@@ -105,29 +135,80 @@ class Preview extends Component {
           <h2>Preview</h2>
           <InfoIcon infoText={previewText} />
         </div>
-        <div className={classNames({ 'Preview-muted': !numFrames })}>
+        <div className="Frame-section-container">
+          <img
+            className="Frame-directional-icon"
+            src={left}
+            onClick={() =>
+              this.handleChangePreviewIdx(Math.max(previewIdx - 1, 0))
+            }
+            alt=""
+          />
           <Frame
             imageSrc={imageSrc}
             playing={playing}
             togglePlaying={this.handleTogglePlaying}
           />
-          <div className="Preview-scrubber" data-testid="Preview-scrubber">
-            <input
-              type="range"
-              min="0"
-              max={numFrames - 1}
-              value={previewIdx}
-              onChange={this.handlePreviewUpdate}
-              disabled={!numFrames}
-              aria-label="preview frame index"
+          <img
+            className="Frame-directional-icon"
+            src={right}
+            onClick={() =>
+              this.handleChangePreviewIdx(
+                Math.min(previewIdx + 1, frameIDs.length - 1)
+              )
+            }
+            alt=""
+          />
+        </div>
+        <div className="Frame-delete">
+          {numFrames ? (
+            <button
+              className="Frame-delete-redo-button"
+              aria-label="delete this frame"
+              onClick={() => this.handleDeleteFrame(previewIdx)}
+            >
+              Delete this Frame
+            </button>
+          ) : null}
+          {redoFrames.length ? (
+            <button
+              className="Frame-delete-redo-button"
+              aria-label="redo last frame"
+              onClick={this.handleRedoFrame}
+            >
+              Undo Delete
+            </button>
+          ) : null}
+        </div>
+        <div className="Preview-scrubber" data-testid="Preview-scrubber">
```

Comment:
I swore I left a comment on this but perhaps I was waiting for the first round of fixes. The final thing to adjust is to put the `.Preview-scrubber-counter` inside of the `.Preview-scrubber` div, right after the slider. Then, place that div _before_ the `.Frame-delete` div.

If not clear, get with me and we implement together.

Reply:
Moved